test/fixtures/syntax/issue-350.json

```diff
@@ -214,10 +254,30 @@
                   "content": [
                     {
                       "element": "number",
+                      "attributes": {
+                        "line": {
+                          "element": "number",
+                          "content": 8
+                        },
+                        "column": {
+                          "element": "number",
+                          "content": 1
+                        }
+                      },
                       "content": 110
                     },
                     {
                       "element": "number",
+                      "attributes": {
+                        "line": {
+                          "element": "number",
+                          "content": 9
+                        },
+                        "column": {
+                          "element": "number",
+                          "content": 1
```

Comment:
As per the spec, the column should be zero-indexed. Since this is empty line this should be 0?

https://api-elements.readthedocs.io/en/latest/element-definitions.html#source-map


---

Also, given the ADD I think the source map should not end on the start of line 9, but end of line 8? Maybe larger issue to deal with than in the scope of this PR though.

Reply:
I use algorithm which was already there for `-l` switch. So it is wrong in original code.
I will look at it

Reply:
Actually, I don't see where in the spec the line or column is said to be zero-indexed. Only the byte offset of the source map is said to be zero-indexed. The `column` and `line` attributes are actually very undefined per spec.

Reply:
Hmm, you are right. We should clarify in the spec. Should we go for 1-indexed or 0-indexed?

Reply:
https://github.com/apiaryio/drafter/pull/653#discussion_r232318730
> column and line numbers are good old natural numbers, and I would keep that. Natural numbers, in my world, start from 1.